Bottom: 56bba892823b5ff4e81d2d91ee59b163db6bc8d2
Top:    16a8eccfe7903b4858be0358527d20581ff30344
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-27 11:31:35 +0100

Refresh of hid-png-increase-the-maximum-a

---

diff --git a/src/hid/png/png.c b/src/hid/png/png.c
index fc75092..bfa98ad 100644
--- a/src/hid/png/png.c
+++ b/src/hid/png/png.c
@@ -169,7 +169,7 @@ HID_Attribute png_attribute_list[] = {
 #define HA_pngfile 0
 
   {"dpi", "Scale factor (pixels/inch). 0 to scale to fix specified size",
-   HID_Integer, 0, 9000, {100, 0, 0}, 0, 0},
+   HID_Integer, 0, 10000, {100, 0, 0}, 0, 0},
 #define HA_dpi 1
 
   {"x-max", "Maximum width (pixels).  0 to not constrain.",
